oam/install_scripts/mcs-controllernode.service.in

```diff
@@ -9,7 +9,7 @@ Environment="SKIP_OAM_INIT=1"
 ExecStart=@ENGINE_BINDIR@/controllernode
 Restart=on-failure
 ExecStop=@ENGINE_BINDIR@/mcs-stop-controllernode.sh
-TimeoutStopSec=$(mcsGetConfig SystemConfig DBRMTimeout)
+TimeoutStopSec=15
```

Comment:
Cannot evaluate during assignment to TimeoutStopSec, not even if /usr/bin/env bash -c "$(mcsGetConfig SystemConfig DBRMTimeout)" is used. Workaround for now, otherwise timeout is 90 seconds by default

Reply:
We don't have any ExecStop commands here so I would replace TimeoutStopSec with a pooling loop in mcs-stop-controllernode with a timeout based on DBRMTimeout + 10.